packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1183,6 +1202,9 @@ export interface IBuildImage {
    */
   readonly repository?: ecr.IRepository;
 
+  /** Optional bind function. */
+  readonly bind?: IBuildImageBind;
```

Comment:
I guess call this a `bindER`? We're in the [kingdom of nouns](http://steve-yegge.blogspot.com/2006/03/execution-in-kingdom-of-nouns.html) after all.

Reply:
Done!